data/coalition outfits.txt

```diff
@@ -116,7 +116,7 @@ outfit "Small Repair Module"
 	"hull repair rate" .12
 	"hull energy" .12
 	"hull heat" .1
-	description "Arachi ships, which rely more on strong hull than on energy shields, often carry one of these repair modules to allow damaged hull plating to be patched back together on the fly."
+	description "Arachi ships, which rely more on strong hull than on energy shields, often carry one of these repair modules to allow damaged hull plating to be patched back together on the fly. You wander if all these outfits will protect the coalition from a fleet of Quarg ships."
```

Comment:
wander -> wonder
Outfit descriptions shouldn't be written like they're story dialog (i.e. they shouldn't have player's thoughts.)